src/openMVG/sfm/pipelines/sequential/sequential_SfM.cpp

```diff
@@ -1027,7 +1027,7 @@ bool SequentialSfMReconstructionEngine::Resection(const size_t viewIndex)
     }
     if(!sfm::SfM_Localizer::RefinePose(
       optional_intrinsic.get(), pose,
-      resection_data, true, b_new_intrinsic))
+      resection_data, true, false))
```

Comment:
@pmoulon Just out of curiosity, do you notice a difference between an early optimization (the root of all evil :-D) and a delayed (intrinsics) optimization during the BA? I was wondering myself what is the impact of having a "good/optimized" solution before doing the triangulation of the new points, and use a "first guess" solution (with no distortion coefficients estimated)  to do triangulation and then optimize/refine everything in the BA. 
One thing that comes to mind, is that with delayed optimization maybe the estimation of the distortion coefficients could be better, because *in theory* you can cover more regions of the image with the new points (while sometimes when you do resection the points could be in one side/region of the image, and thus if u refine just after that you may bias a bit the estimation of the distortion (or not, i really don't know)). But of course it depends on images etc.
On the other hand it's true that triangulation without undistorted points may introduce some (or a lot of noise), especially if the points are far from the image center. And I don't know if BA can fix that later.
Did you notice some effect like that in your experiments?